Resources/translations/FOSUserBundle.cs.yml

```diff
@@ -23,38 +25,60 @@ profile:
         submit: Upravit
     flash:
         updated: 'Profil byl úspěšně aktualizován'
+
 change_password:
     submit: 'Nastavit nové heslo'
     flash:
         success: 'Nové heslo bylo úspěšně nastaveno'
+
 registration:
-    check_email: 'Na adresu %email% byl zaslán aktivační e-mail. Obsahuje aktivační odkaz, na který je nutné kliknout pro aktivaci Vašeho účtu.'
+    check_email: 'Na adresu %email% byl zaslán aktivační e-mail. Obsahuje odkaz, na který je nutné kliknout pro aktivaci Vašeho účtu.'
     confirmed: 'Gratulujeme, %username%, Váš účet je nyní aktivní.'
     back: 'Návrat na předchozí stránku.'
     submit: 'Registrovat se'
     flash:
         user_created: 'Váš účet byl úspěšně vytvořen'
     email:
         subject: 'Vítejte, %username%!'
-        message: "Dobrý den, %username%!\n\nPro dokončení Vaší registrace klikněte na tento odkaz: %confirmationUrl%\n\nS pozdravem,\nrealizační tým.\n"
+        message: |
+            Dobrý den, %username%!
+            
+            Pro dokončení Vaší registrace prosím klikněte na tento odkaz: %confirmationUrl%
+            
+            S pozdravem,
+            realizační tým.
+
 resetting:
     password_already_requested: 'Tento uživatel již zažádal o nastavení nového hesla (během posledních 24 hodin).'
-    check_email: 'Na adresu %email% byl zaslán e-mail s návodem na nastavení nového hesla. E-mail obsahuje odkaz, na který je nutné kliknout, bez tohoto úkonu není možné nové heslo nastavit.'
+    check_email: |
+        Byl Vám zaslán e-mail s návodem na nastavení nového hesla. E-mail obsahuje odkaz, na který je nutné kliknout pro nastavení nového hesla.
+        O nastavení nového hesla můžete žádat pouze každých %tokenLifetime% hodin.
+        
+        Pokud jste e-mail neobdrželi, zkontrolujte prosím složku se spamem nebo zkuste obnovování hesla zopakovat.
     request:
         username: 'Uživatelské jméno nebo e-mail'
         submit: 'Nastavit nové heslo'
     reset:
-        submit: 'Nastavit nové heslo'
+        submit: 'Změnit heslo'
     flash:
         success: 'Heslo bylo úspěšně změněno'
     email:
-        subject: 'Zaslání zapomenutého hesla'
-        message: "Dobrý den, %username%!\n\nPro nastavení nového hesla klikněte na tento odkaz: %confirmationUrl%\n\nS pozdravem,\nrealizační tým.\n"
+        subject: 'Nastavení nového hesla'
+        message: |
+            Dobrý den, %username%!
+            
+            Pro nastavení nového hesla prosím klikněte na tento odkaz: %confirmationUrl%
+            
+            S pozdravem,
+            realizační tým.
+
+# Globální výrazy
```

Comment:
It's an old comment that doesn't really add much value. It would remove both comments.